datavec/datavec-python/src/main/java/org/datavec/python/PythonExecutioner.java

```diff
@@ -62,23 +62,36 @@ private static synchronized void init() {
         numpy._import_array();
     }
 
-    private static synchronized void _exec(String code) {
+    private static synchronized void _exec(String code) throws PythonException{
```

Comment:
While we're changing things, please get rid of the pythonic under score prefixes. They aren't needed in java. That's what the explicit scoping is for.